src/components/Petitions/index.js

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import PetitionListItem from 'components/PetitionListItem';
 
 const Petitions = ({ total, petitions, currentPage, perPage }) => (
-  <ul>
-    {petitions.map((petition) => <PetitionListItem {...petition} key={petition.id} />)}
-  </ul>
+  <div>
+    <h1>Petitionen</h1>
```

Comment:
Ideally should come from [`config.js`](https://github.com/iris-dni/iris-frontend/blob/develop/src/settings/config.js) for now, before we add translation handling. Give as 'Petitions' in English for IRIS, then we over-write for Petitio.ch in the custom config.